packages/core/storybook/Breadcrumbs.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+import { Breadcrumbs, Icon } from '../src'
+
+storiesOf('Breadcrumbs', module).add('Basic', () => (
+  <Breadcrumbs>
+    <Breadcrumbs.Link
+      href="https://www.priceline.com"
+      label="Home"
+      icon={<Icon color="text.light" size={16} m={2} name="Home" />}
```

Comment:
Nitpick: Use specific icons instead of generic Icon with `name` prop.
Example:
`<HomeIcon size={16} m={2} />`

Reply:
For sure! Updated the storybook and test as well.

Reply:
I also added a plain version with no icons to the storybook/tests (please see updated description)!